src/dash-playlist-loader.js

```diff
@@ -728,7 +731,7 @@ export default class DashPlaylistLoader extends EventTarget {
       // If we haven't yet selected a playlist, wait until then so we know the
       // target duration
       if (!this.media()) {
-        this.one('loadedplaylist', () => {
+        this.one('loadedmetadata', () => {
```

Comment:
When media loads on the master playlist for the first time it's called `loadedmetadata`. `loadedplaylist` signifies the load of the master playlist.